### PR TITLE
feature #833: created an endpoint for bot to create events

### LINKED ISSUE #833: Create an Endpoint for bot to create Events
Your task is to create an **Endpoint** for bot to create an **Event** _(without of labs, practices, lectures)_

**Body:**
```typescript
{
  "groupId": "string",
  "name": "string",
  "startTime": "2023-11-04T20:55:47.931Z",
  "endTime": "2023-11-04T20:55:47.931Z",
  "period": "NO_PERIOD",
  "url": "string",
  "eventInfo": "string"
}
```

_Have fun with this task)_

### PR DESCRIPTION
**Pull request by ArtemiiLiakh**
- created endpoint createSimpleTelegramEvent that creates simple version of an event
- created mapper method for the endpoint
- created dto for the endpoint
- created respoce for the endpoint
- added documentation for the endpoint

closses #833

### CODE REVIEW

File: src/v2/mappers/ScheduleMapper.ts [hunk 1]
```diff
@@ -57,6 +57,18 @@ export class ScheduleMapper {
...
 
+  getSimpleEvent (event: DbEvent): SimpleEventResponse {
+    return {
+      id: event.id,
+      name: event.name,
+      startTime: event.startTime,
+      endTime: event.endTime,
+      period: event.period,
+      url: event.url || null,
+      eventInfo: event.eventInfo[0]?.description || null,
```
Comment (f-ogoog) [inline src/v2/mappers/ScheduleMapper.ts:68]: now get endpoints return events without eventInfo, so you need improve 3 endpoints for get day/week/fortnight events and add field eventInfo

I think you can just add this field in getTelegramEvents

File: src/v2/mappers/ScheduleMapper.ts [hunk 2]
```diff
@@ -57,6 +57,18 @@ export class ScheduleMapper {
...
   }
 
+  getSimpleEvent (event: DbEvent): SimpleEventResponse {
+    return {
+      id: event.id,
+      name: event.name,
+      startTime: event.startTime,
+      endTime: event.endTime,
+      period: event.period,
+      url: event.url || null,
```
Comment (f-ogoog) [inline src/v2/mappers/ScheduleMapper.ts:67]: needless null

**Review by f-ogoog — CHANGES_REQUESTED** [review]

File: src/v2/mappers/ScheduleMapper.ts [hunk 3]
```diff
@@ -57,6 +58,17 @@ export class ScheduleMapper {
...
   }
 
+  getSimpleEvent (event: DbEvent): SimpleEventResponse {
+    return {
+      id: event.id,
+      name: event.name,
+      startTime: event.startTime,
+      endTime: event.endTime,
+      period: event.period,
+      url: event.url,
```
Comment (f-ogoog) [inline src/v2/mappers/ScheduleMapper.ts:68]: you should add eventInfo and use this finction is getTelegramEvents

File: src/v2/api/responses/EventResponse.ts [hunk 4]
```diff
@@ -54,4 +54,37 @@ export class EventResponse {
     type: [TeacherNamesResponse],
   })
     teachers: TeacherNamesResponse[];
+}
+
+export class SimpleEventResponse {
```
Comment (f-ogoog) [inline src/v2/api/responses/EventResponse.ts:59]: needless class, you should use TelegramEventInfoResponse

**Review by f-ogoog — CHANGES_REQUESTED** [review]

**Review by f-ogoog — APPROVED** [review]

**Review by illiakroshka — CHANGES_REQUESTED** [review]
The example of responce contains "disciplineType"
image

But actual responce doen't have it 
image

**Review by illiakroshka — APPROVED** [review]

**Review by fokaaas — APPROVED** [review]

**Pull request merged**